pkg/client/unversioned/request_test.go

```diff
@@ -729,6 +733,44 @@ func TestDoRequestNewWay(t *testing.T) {
 	fakeHandler.ValidateRequest(t, requestURL, "POST", &reqBody)
 }
 
+// This test assumes that the client implementation backs off exponentially, for an individual request.
+func TestBackoffLifecycle(t *testing.T) {
+	count := 0
+	os.Setenv(envBackoffBase, "1")
+	os.Setenv(envBackoffDuration, "120")
```

Comment:
I think  you are right: The purpose isn't to test env vars, but to test lifecycle... But the other comment below i am a little concerned with - we need some coverage somewhere.    In any case, i  will update this part accordingly.